core/Pos/Binary/Class.hs

```diff
@@ -58,7 +58,10 @@ module Pos.Binary.Class
        , biSize
        ) where
 
+import           Universum
```

Comment:
If we want to put `Universum` import into a separate group, it should be mentioned in style guide. Currently I don't see such rule there. According to current version of style guide it should be put into the first group together with everything from other hackage packages.